### PR TITLE
README update

### LINKED ISSUE #64: README needs an update
It still mentions custom CSS, which is no longer part of journal.

### PR DESCRIPTION
**Pull request by hydeenoble**
Fixes #64 

cc @lukebarnard1 for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ This will build journal using [browserify](http://browserify.org/) and will watc
 The `dev` script is enough to deploy, and requires node with Express installed. Otherwise, any other web server is perfectly fine -- when built, journal is simply a collection of files to be served over HTTP.
 
 #### Style Modifications
-Overriding CSS can be placed in the CSS style sheet located in [custom.css](./web/custom.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
+Overriding CSS can be placed in the CSS style sheet located in [style.css](./web/style.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
```
Comment (lukebarnard1) [inline README.md]: Actually this is wrong; it should say "To modify the style of the app, the CSS in style.css can be changed."  no extra steps need to be taken, so you can remove the bit that says

>To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html).  Alternatively, the main style sheet [style.css](./web/style.css) can be edited.

Comment (hydeenoble) [reply README.md]: are you saying 

```
Overriding CSS can be placed in the CSS style sheet located in [style.css](./web/style.css). To include it in the running version of journal, uncomment the relevant line in the `<head>` tag of [index.html](./web/index.html). Alternatively, the main style sheet [style.css](./web/style.css) can be edited.
```

should be replace with

```
To modify the style of the app, the CSS in [style.css](./web/style.css) can be changed.
```

Comment (lukebarnard1) [reply README.md]: yep :)

File: package.json [hunk 2]
```diff
@@ -1,33 +0,0 @@
-{
```
Comment (lukebarnard1) [inline package.json]: This should not be removed.

Comment (hydeenoble) [reply package.json]: Apologies, that was a mistake on my part.... on it

**Comment by lukebarnard1** [comment]
Closing in favour of https://github.com/lukebarnard1/journal/pull/66

**Pull request closed without merging**